src/main.cpp

```diff
@@ -2070,9 +2070,9 @@ void usageBus() {
        << "    --rf-stranded             Strand specific reads for UMI-tagged reads, first read reverse" << endl
        << "    --unstranded              Treat all read as non-strand-specific" << endl
        << "    --paired                  Treat reads as paired" << endl
-       << "    --long                  	Treat reads as long" << endl
+       << "    --long                    Treat reads as long" << endl
        //<< "    --error_rate              Estimated error rate of long reads (required for --long)" << endl
-       << "    --threshold		            Threshold for rate of unmapped kmers per read" << endl
```

Comment:
This looks accidental.

Reply:
Could you clarify what looks accidental?

Reply:
According to the diff on github, you are removing whitespace here:
![image](https://github.com/pachterlab/kallisto/assets/12913701/a4f89559-835a-49bd-bc1f-697acbbe9b2d)
In line 2073 you fix the alignment of 'Treat' but in line 2075, 'Threshold' is shifted to left too much.